pyproject.toml

```diff
@@ -3,21 +3,7 @@ line-length = 79
 exclude = '''
 (
   /(
```

Comment:
If I understand correctly, this for for excluding certain folders from being checked with `black`. We are removing these because the `tox` will run the `eachdist.py` script (which will only run on folders specified in `eachdist.ini`), instead of running `black` directly correct? For cases when I want to fix the lint directly myself without running `tox`, will changing this file effect that?

Reply:
No, running black directly vs eachdist.py makes no difference at all. This was removed because black already checks .gitignore and so the lines were just redundant. This is actually a change independent from the rest of this PR and could be merged separately.